src/data/clients.ts

```diff
@@ -637,6 +637,30 @@ export const Clients: Array<Client> = [
     ],
     recommended: true
   },
+  {
```

Comment:
New entries should be added at the end.

Reply:
Thank for the feedback. I have moved it to the end.